src/lib/Api.ts

```diff
@@ -103,7 +103,8 @@ class Collection {
                 },
                 body: {
                     query,
-                    projection
+                    projection,
+                    options
```

Comment:
Handle default value (if needed):

```suggestion
                    projection ?? {},
                    options ?? {},
```